src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc

```diff
@@ -744,234 +744,80 @@ Defines the display element from the selected report the page block will use whe
 
 |===
 
-[#page-block-geometry]
-== Geometry
+[#page-block-tabs]
+== Tabs
 
-The *geometry page block* allows you to show a map with markers, lines and polygons from different geometry values of different record sources.
+The *tabs* page block allows the grouping of different page blocks together, decluttering, and making it easier to navigate a page.
 
-image:compose-configuration/page-block-reference/config-geometry.png[...,49%,role="preview data-zoomable"]
-image:compose-configuration/page-block-reference/added-geometry.png[...,49%,role="preview data-zoomable"]
+=== Configuring Tabs
 
-.Additional configurations:
-[cols="2s,5a"]
-|===
-| [#page-block-zoom-min]#<<page-block-zoom-min,Zoom min>>#
-|
-Defines the minimum zoom level of the map.
-
-| [#page-block-zoom-max]#<<page-block-zoom-max,Zoom max>>#
-|
-Defines the maximum zoom level of the map.
-
-| [#page-block-zoom-lockbounds]#<<page-block-zoom-lockbounds,Lock bounds>>#
-|
-Defines if the map should be prevented from being panned out or zoomed out of the current rectangular area of the map, specified by its longitude and latitude coordinates. This means that the map will only display the area within the bounds of the specified coordinates and prevent the user from panning or zooming out beyond those bounds.
-
-|===
-
-[#page-block-navigation]
-== Navigation
-
-The *navigation page block* allows you to define custom navigation inside your {PRODUCT_NAME} {APP_NAME_COMPOSE} applications applications. The navigation can be an addition to the pre-defined one or a complete replacement.
-
-image:compose-configuration/page-block-reference/config-navigation.png[...,49%,role="preview data-zoomable"]
-image:compose-configuration/page-block-reference/added-navigation.png[...,49%,role="preview data-zoomable"]
-
-.Additional configurations:
-[cols="2s,5a"]
-|===
-| [#page-block-navigation-appearance]#<<page-block-navigation-appearance,Appearance>>#
-|
-Defines the style of the navigation
-
-.Supported appearances:
-* Tabs +
+To tab a page block, select the desired page block from the list of available page blocks.
 
+.Tabs page block configuration:
 [annotation,role="data-zoomable"]
 ----
 {
   "image": {
-    "rel": "compose-configuration/page-block-reference/added-navigation-tabs.png",
-    "alias": "compose-configuration-page-block-reference-added-navigation-tabs",
+    "rel": "compose-configuration/config-tabs-add-button.png",
```

Comment:
The blur is still off; please tweak it.
Also, I'm not sure the annotations you've added match up... They don't make sense in my opinion so I'd remove it.